src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -455,31 +455,6 @@ static IEnumerable<Location> GetUnnecessaryLocations(Diagnostic diagnostic)
             }
         }
 
-        private static void TestDiagnosticTags(
```

Comment:
This doesn't have anything to do with the commit character PR - I somehow just had this file open and noticed this unused method.